src/components/Main/Header.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react';
+import Logo from '../../images/logo.png';
+import { NavLink, useNavigate } from 'react-router-dom';
+import useCookie from '../../hooks/useCookie';
+
+function Header () {
+  const navigate = useNavigate();
+
+  const {
+    getCookie,
+    state: { isLoggedIn }
+  } = useCookie();
+
+  const navMenu = [
+    { path: '/', title: 'Home' },
+    { path: '/products', title: 'Shop' },
+    { path: '/contact', title: 'Contact' }
+  ];
+
+  const [isActive, setIsActive] = useState(window.location.pathname);
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const search = (searchTerm) => {
+    navigate(`/products/filter?name=${searchTerm}`);
```

Comment:
Lets add some dropdown searchComponent instead of navigating to separate products page.